#! /bin/sh /usr/share/dpatch/dpatch-run
## fix_ssrc_change.dpatch by  <root@>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix ssrc change

@DPATCH@

--- linphone-2.1.1~/mediastreamer2/src/audiostream.c	2011-06-12 08:33:48.000000000 +0000
+++ linphone-2.1.1/mediastreamer2/src/audiostream.c	2011-06-12 08:49:02.000000000 +0000
@@ -114,6 +114,8 @@
 	rtp_session_enable_adaptive_jitter_compensation(rtpr,TRUE);
 	rtp_session_set_symmetric_rtp(rtpr,TRUE);
 	rtp_session_set_local_addr(rtpr,ipv6 ? "::" : "0.0.0.0",locport);
+        rtp_session_signal_connect(rtpr,"timestamp_jump",(RtpCallback)rtp_session_resync,(long)NULL);
+        rtp_session_signal_connect(rtpr,"ssrc_changed",(RtpCallback)rtp_session_resync,(long)NULL);
 
 	return rtpr;
 }
